### ISSUE TITLE
Kit installation?

### ISSUE THREAD

**Issue by bradparks** [KTXSoftware/Kit.app#1]
Hey! Just a quick note - I didn't realize at first I had to recursively clone Kit, and then once I did and ran it, it gives me a "Hake error" when I "Download", "Update" and try to "Open" then "Create" any version of the Empty project (eg Flash). Thanks!

Note that I wouldn't have had any idea how to do this other than following the Khapunk readme here:
https://bitbucket.org/stalei/khapunk/wiki/Gettingstarted.md

![screen shot 2015-10-05 at 10 38 57 pm](https://cloud.githubusercontent.com/assets/185477/10298177/ebed44d8-6bb1-11e5-92a2-da276b54c496.png)

**Comment by bradparks** [comment]
Note that creating a project seems to work fine if I just clone Empty into my projects working directory instead....

**Comment by RobDangerous** [comment]
Uh oh, didn't work on Kit for a while. Guess it's about time to get back to it.

**Comment by bradparks** [comment]
yeah I'd never heard of it before, but wanted to try and use Khapunk, and it led me down that path, which seems legit, as it documented the basic process...
